src/js/apis/RespecterCestProtegerService.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+
+function RespecterCestProtegerService() {
+    this.axios = axios.create()
+}
+
+RespecterCestProtegerService.prototype.identify = function(position, extent, mapWidth, mapHeight, language) {
+    if (language !== 'fr' && language !== 'de' && language !== 'it') {
+        language = 'en'
+    }
+
+    const baseUrl = 'https://api3.geo.admin.ch/rest/services/api/MapServer/identify';
```

Comment:
semi ;) 

By the way, it's probably the good moment to switch to good semi config + 2 spaces indents. Let me know your opinion.

Reply:
Right 👍 
As for switching to mandatory semi colon and 2 spaces indent, do it whenever you are ready for it. I can quite easily update my PR to match this (there are already conflicts anyway)

Reply:
ok